.env.example

```
@@ -0,0 +1,5 @@
+# Clerk Authentication
+EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_your_publishable_key_here
+
+# Convex Backend
+CONVEX_DEPLOYMENT=your_convex_deployment_url_here
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)